tests/test_routes.py

```diff
@@ -0,0 +1,5 @@
+
+
+def test_index_page(flask_test_client):
+    response = flask_test_client.get("/")
+    assert response.status_code == 302
```

Comment:
Because we have an index page that redirects to the upload page because we don't currently have anything to populate the index page with. I followed how we set up it for find.

Reply:
Also added a test to verify that /upload returns a 200 status code.